csi.proto

```diff
@@ -1056,6 +1056,8 @@ message ControllerServiceCapability {
       // If for some reason Controller and Node Plugins report
       // misaligned volume conditions, CO SHALL assume the worst case
       // is the truth.
+      // Note that `VolumeCondition` can be used to generate events and
```

Comment:
"events" .. smells like k8s lingo.

also, I specifically called out use of the message field of VolumeCondition. and how it's not clear what the intended use is for it. can you please attempt to clarify? we cannot assume k8s semantics/terminology in this spec - it should be very clear to all users of all COs what `message` means. and whether it is for humans or machines to interpret